src/hash.h

```diff
@@ -27,51 +27,51 @@ enum {
 	} cipher##_##bits##_context_t;\
```

Comment:
my astyle changed the formatting of this file, not sure if I have the settings wrong or the file was just not formatted properly previously